cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/GcpResourceConnector.java

```diff
@@ -60,8 +60,8 @@ public String getStackTemplate() throws TemplatingDoesNotSupportedException {
     }
 
     @Override
-    public String getDBStackTemplate() {
-        return  "";
+    public String getDBStackTemplate() throws TemplatingDoesNotSupportedException {
```

Comment:
does --> is

Reply:
renamed to TemplatingNotSupportedException